src/diffusers/utils/dummy_torch_and_transformers_objects.py

```diff
@@ -32,6 +32,21 @@ def from_pretrained(cls, *args, **kwargs):
         requires_backends(cls, ["torch", "transformers"])
 
 
+class AudioLDMPipeline(metaclass=DummyObject):
+    _backends = ["torch", "transformers"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["torch", "transformers"])
```

Comment:
Usually we gate the import on the transformers version that dependencies are in 

https://github.com/huggingface/diffusers/blob/54bc882d96db1d581f66883cde8f2a5c7eccb919/src/diffusers/pipelines/stable_diffusion/__init__.py#L61-L67

I think that means we can just gate on the next transformers patch release number. Even if the next release is a minor release, it will still be greater than the next patch release. cc @patrickvonplaten , any objections with gating on a not yet released version of transformers?

Reply:
Resolved in https://github.com/huggingface/diffusers/pull/2232/commits/9f776a2845abd457a6e471e1ef4aacb4c9005ba9, currently requires Transformers v4.27.0 (the next minor release).

Reply:
v4.27.0 released yesterday: https://github.com/huggingface/transformers/releases/tag/v4.27.0